Makefile

```diff
@@ -2,6 +2,8 @@
 # --------------------------
 REGISTRY?=gcr.io/k8s-staging-metrics-server
 ARCH?=amd64
+# The output type could either be docker (local), or registry.
+OUTPUT_TYPE ?= docker
```

Comment:
This way `docker buildx build --output=type=xxx --platform xxx` is ok for me
I see the comments here [OUTPUT_TYPE](https://github.com/kubernetes/kubernetes/blob/master/build/pause/Makefile#L31) , the value `registry` of `OUTPUT_TYPE` is used for windows images, I'm not sure if this parameter needs to be added, need more discussion.